src/viur/core/render/html/env/viur.py

```diff
@@ -570,6 +573,7 @@ def renderEditForm(render: Render,
     sectionTpl = render.getEnv().get_template(render.getTemplateFileName("editform_category"))
     rowTpl = render.getEnv().get_template(render.getTemplateFileName("editform_row"))
     sections = OrderedDict()
+
```

Comment:
Can we add a check like this before?

```suggestion

if both := set(ignore).intersection(bones):
    raise ValueError(f"You have specified the same bones {', '.join(both)} in *ignore* AND *bones*!")
```